_includes/codepen.html

```diff
@@ -1,5 +1,5 @@
 <script type="text/javascript">
-  var currentScript = 'https://unpkg.com/fabric@4.3.0/dist/fabric.js';
+  var currentScript = 'https://unpkg.com/fabric@latest/dist/fabric.js';
```

Comment:
great!

Reply:
😉

Reply:
BTW I was thinking of the CDN. Can it accept params?
`https://unpkg.com/fabric@latest/dist/fabric.js?include=erasing`